package.json

```diff
@@ -25,132 +25,133 @@
     "font-awesome": "^4.7.0",
     "glob": "^7.1.1",
     "history": "^4.7.2",
-    "http-link-header": "^0.8.0",
+    "http-link-header": "^1.0.2",
     "immutable": "^3.8.2",
     "imports-loader": "^0.8.0",
-    "intersection-observer": "^0.5.0",
+    "intersection-observer": "^0.5.1",
     "intl": "^1.2.5",
     "intl-messageformat": "^2.2.0",
     "intl-relativeformat": "^2.1.0",
     "is-nan": "^1.2.1",
     "js-yaml": "^3.11.0",
-    "lodash": "^4.17.5",
+    "lodash": "^4.7.11",
     "mark-loader": "^0.1.6",
-    "marky": "^1.2.0",
-    "mini-css-extract-plugin": "^0.4.2",
+    "marky": "^1.2.1",
+    "mini-css-extract-plugin": "^0.5.0",
     "mkdirp": "^0.5.1",
-    "node-sass": "^4.9.2",
     "npmlog": "^4.1.2",
     "object-assign": "^4.1.1",
     "object-fit-images": "^3.2.3",
     "object.values": "^1.0.4",
-    "offline-plugin": "^5.0.5",
+    "offline-plugin": "^5.0.6",
     "path-complete-extname": "^1.0.0",
     "pg": "^6.4.0",
     "postcss-loader": "^3.0.0",
     "postcss-object-fit-images": "^1.1.2",
-    "postcss-smart-import": "^0.7.6",
-    "precss": "^3.1.2",
     "prop-types": "^15.5.10",
     "punycode": "^2.1.0",
-    "rails-ujs": "^5.2.0",
-    "react": "^16.5.0",
-    "react-dom": "^16.5.0",
-    "react-hotkeys": "^0.10.0",
+    "rails-ujs": "^5.2.2",
+    "react": "^16.7.0",
+    "react-dom": "^16.7.0",
+    "react-hotkeys": "^1.1.4",
     "react-immutable-proptypes": "^2.1.0",
     "react-immutable-pure-component": "^1.1.1",
-    "react-intl": "^2.4.0",
+    "react-intl": "^2.7.2",
     "react-motion": "^0.5.2",
     "react-notification": "^6.8.4",
     "react-overlays": "^0.8.3",
-    "react-redux": "^5.0.4",
-    "react-redux-loading-bar": "^2.9.3",
+    "react-redux": "^6.0.0",
+    "react-redux-loading-bar": "^4.1.0",
     "react-router-dom": "^4.1.1",
     "react-router-scroll-4": "^1.0.0-beta.1",
-    "react-select": "^2.0.0",
+    "react-select": "^2.2.0",
     "react-sparklines": "^1.7.0",
-    "react-swipeable-views": "^0.12.17",
-    "react-textarea-autosize": "^5.2.1",
+    "react-swipeable-views": "^0.13.0",
+    "react-textarea-autosize": "^7.1.0",
     "react-toggle": "^4.0.1",
     "redis": "^2.7.1",
-    "redux": "^3.7.1",
+    "redux": "^4.0.1",
     "redux-immutable": "^4.0.0",
     "redux-thunk": "^2.2.0",
-    "rellax": "^1.6.2",
+    "rellax": "^1.7.1",
     "requestidlecallback": "^0.3.0",
-    "reselect": "^3.0.1",
+    "reselect": "^4.0.0",
     "rimraf": "^2.6.1",
+    "sass": "^1.15.2",
     "sass-loader": "^7.0.3",
-    "stringz": "^0.3.0",
-    "style-loader": "^0.23.0",
+    "stringz": "^1.0.0",
+    "style-loader": "0.23.1",
     "substring-trie": "^1.0.2",
     "throng": "^4.0.0",
     "tiny-queue": "^0.2.1",
-    "uglifyjs-webpack-plugin": "^1.2.7",
+    "uglifyjs-webpack-plugin": "^2.1.1",
     "uuid": "^3.1.0",
     "uws": "10.148.0",
-    "webpack": "^4.18.0",
-    "webpack-bundle-analyzer": "^2.13.1",
-    "webpack-cli": "^3.0.8",
-    "webpack-manifest-plugin": "^2.0.4",
```

Comment:
Replace to `webpack-assets-manifest`.

`webpack-manifest-plugin` has not been maintained for a long-term.